src/runtime/resources/clr.py

```diff
@@ -2,7 +2,7 @@
 Code in this module gets loaded into the main clr module.
 """
 
-__version__ = "2.5.0"
+__version__ = "2.5.1"
```

Comment:
Please exclude the 2.5.1 commits. The actual changes from that branch should already be in master (if not, make them a separate PR, the update to the Changelog makes sense).